src/tests/system/tests/test_passkey.py

```diff
@@ -0,0 +1,495 @@
+from __future__ import annotations
+
+import pytest
+import os
+
+from lib.sssd.roles.client import Client
+from lib.sssd.roles.ipa import IPA
+from lib.sssd.roles.ldap import LDAP
+from lib.sssd.topology import KnownTopology
+
+
+@pytest.mark.tier(0)
+@pytest.mark.topology(KnownTopology.Client)
+def test_passkey__sssctl_register(client: Client):
+    """
+    :title: Register a key with sssctl
+    :setup:
+        1. Setup IDM client with FIDO and umockdev setup
+    :steps:
+        1. Use sssctl to register a FIDO2 key.
+        2. Check umockdev-run that the output contains the user key mapping data.
+    :expectedresults:
+        1. New key is registered
+        2. Output contains key mapping data.
+    :customerscenario: False
+    """
+    tc = "sssctl_reg_test1"
+    cmd = "sssctl passkey-exec --register --username=user1 --domain=ldap.test"
+
+    for record_file in ['yk.umockdev', f'{tc}.ioctl', f'{tc}.script', f'{tc}.output']:
+        src_file = f"{os.getcwd()}/data/passkey/{tc}/{record_file}"
+        client.fs.upload(src_file, f"/tmp/{record_file}")
```

Comment:
Move the copy to a function and reuse it for all the tests

Reply:
yes added the method to copy.